bin/init.gradle

```diff
@@ -24,6 +24,17 @@ allprojects { everyProj ->
 
                         cp.add(v.javaCompile.destinationDir.getAbsolutePath())
                     }
+
+                    // apply config-attributes in classpath recovery
+                    if (project.hasProperty('confAttrs')) {
+                        def confAttrSpec = (project.hasProperty('confAttrs') 
+                            ? confAttrs.toLowerCase().split(',').collectEntries { [(it.split(':')[0]) : it.split(':')[1]] }
+                            : null
+                        )
+
+                        cp.unique().removeIf { (it.toLowerCase() ==~ /.*${confAttrSpec['backend']}(?!.*${confAttrSpec['buildtype']}).*/) }
```

Comment:
Is `backend` used for your specific usage?

Reply:
yes. This is the specific flavorDimention defined in our `build.gradle.kts` config file. During the integration for the Android project, @lili2311 pointed me to set the `backend` variable. However, the backend seems to be a keyword to set the proper `productFlavor`.